starter-code/french-onion-soup-starter-code/css/main.css

```diff
@@ -5,5 +5,57 @@ h1, body, html, main {
 /*Import the font and set the default size*/
 html {
   font-family: 'Open Sans', sans-serif;
-  font-size: 18px
+  font-size: 17px
 }
+
+.center {
+  margin: 0 auto; 
+  width: 700px;
+}
+
+img {
+  display: block;
+  width: 350px;
+  margin: 0 auto;
+}
+
+header {
+  height: 650px;
```

Comment:
by adding a class name to your header, you could have just used one CSS sheet which helps to keep your code dry.  For example : 

```
header .soup {
background-image: url("../images/soup.jpg");
}
```

Most of the css should be the same